config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -506,6 +506,9 @@ for tests in non-alpha alpha; do
       args:
       - ./.prow.sh
       env:
+      # TODO: change to v0.6.0
```

Comment:
why is there a TODO on all of these

Reply:
Using a specific commit of kind is a workaround until kind releases a new version. Updated with bug reference